GOVERNANCE.md

```diff
@@ -1,172 +1,73 @@
-      This should be rare, due to the social cost of discontinuing the Consensus process for this reason.
-      Most decisions should wait for the above process to take its course.
-- If Deciders agree to a vote, the default is a Simple Majority.
-- However, there are cases that require stronger voting – Supermajority or Unanimity – specified below:
-
-### Simple Majority Decisions
-
-If a vote is called, the default is a Simple Majority Vote <https://en.wikipedia.org/wiki/Majority>.
+- Engage deeply in the work being done by the working group - that is, the role is not purely administrative.
+  Chairs are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
+- When necessary, handle human and technical coordination across different working group workstreams
 
-### Supermajority Decisions
+### Current Chairs
 
-If a vote is called, the following decisions require a Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority>.
+Name | Organization | GitHub
+-- | -- | --
+Dan Garfield | Codefresh | @todaywasawesome
+Leonardo Murillo | Cloud Native Architects | @murillodigital
+Scott Rigby | Weaveworks | @scottrigby
 
-- Oversight Committee: Enforcing a Code of Conduct violation by a community member.
-- Oversight Committee: Licensing and intellectual property changes.
-- Oversight Committee: Material changes to the Governance document.
-  - Note: editorial changes to governance may be made by lazy consensus, unless challenged.
-    These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
-    They do not change the intention or meaning of anything in this document.
+## Elections
 
-### Unanimity Decisions
+[Chairs](#chairs) will be elected for one year terms and may run for reelection.
 
-If a vote is called, the following decisison require Unanimity <https://en.wikipedia.org/wiki/Unanimity>.
+The election process takes place on the CNCF SIG App Delivery [mailing list](cncf-sig-app-delivery@lists.cncf.io ).
 
-- Repository Maintainers: Electing new Maintainers of the same repository.
-- Oversight Committee: Electing new Committee members.
-- Oversight Committee: Removing a Repository Maintainer or Committee member for any reason other than inactivity.
+Before each election, there will be a call for nominiations on the mailing list.
+Nominations should be sent to the mailing list, and include a brief bio and personal statement describing the candidates qualifications to serve in this capacity.
+Self-nominations are welcome.
 
-## Proposal Process
+The GitOps WG employs "organization voting" to ensure no single organization can dominate the election process or project.
+Each organization listed in the [interested parties](interested-parties.md) document has one vote.
```

Comment:
"Up to two individuals from organizations who are both active members of the CNCF and listed in the [interested parties](interested-parties.md) may vote."